### ISSUE TITLE
SSHX plugin - Add timestamp annotation?

### ISSUE THREAD

**Issue by kmorrisCAL** [cronicle-edge/cronicle-edge#97]
Hello,
Thank you for your time and efforts on this wonderful project.  Before the arrival of the SSHX plugin, I was using a combination of `scp` and other bash hackery to do something similar with the shell plugin.

Using the SSHX plugin is much cleaner, but one thing I miss from the shell plugin is the timestamp annotation feature in the log output.  Would it be simple/possible to get this ability added to the SSHX plugin as well?

Thank you.

**Comment by mikeTWC1984** [comment]
Sure, I think that should be fairly easy to add. Will include on the next release

**Comment by kmorrisCAL** [comment]
Excellent.  Much appreciated!

**Comment by kmorrisCAL** [comment]
Hello,
I just pulled down the latest v1.9.0 release and tested this new timestamp annotation functionality.  Perhaps I messed something up in the "migration" -- I had to manually add the `annotate` parameter to the SSHX plugin:
![image](https://github.com/cronicle-edge/cronicle-edge/assets/28564862/01b0faa6-2621-463e-8d38-a732de890134)

After checking the box on an SSHX script, I now see the timestamp info in the log output, but every line has the same timestamp.  Is that expected?

Example script:
![image](https://github.com/cronicle-edge/cronicle-edge/assets/28564862/6aa6586e-4523-4f46-9aba-35fdf9c1d486)

Example log output:
![image](https://github.com/cronicle-edge/cronicle-edge/assets/28564862/62dc8996-c2fa-4f8a-8fc4-472cb74c3d41)

Thank you!
